### DISCUSSION TITLE
日企用交友App 為員工找真愛

### DISCUSSION THREAD

**Discussion by zixunn** [1101-GS4719-C/Python-Community-GS4719-C#324] [閒聊八卦]
與異性聊天有AI當軍師感覺會更有信心一點呢 
[人工智慧（AI）交友軟體](https://udn.com/news/story/6812/6021412)

**Comment by mesakiyuta** [comment]
看到約會成功率高達76%是有點嚇到，有AI來當軍師感覺很可靠。

**Comment by ChinChun0220** [comment]
我比較好奇的是AI是透過那個資料判斷對方興趣呢🤔如果是對方輸入過的興趣還好，還是在上面聊天紀錄提過的東西呢？還是我太陰暗XD

**Reply by ChinChun0220** [reply]
突然想到道德倫理上的問題了，那這樣讓對方有好感成功約出去，有好感的真的是那個人嗎，還是其實是下建議的AI…. 細思極恐 怕.jpg

**Comment by ChinChun0220** [comment]
題外話 一個月6000日圓 台幣1500左右真的不低欸，還是台灣國內的社交軟體也是這個價格？

**Comment by cody910120** [comment]
不是ㄟ我覺得交友軟體踩到地雷機率很高ㄟ，maybe會有幾個真的遇到的案例，可適用交友軟體幫員工找真愛也是偏難吧

**Comment by 107602505** [comment]
AI提供意見應該是在幫助那些不擅長交友的人，教他們如何開起對方可能有興趣的話題並且能持續聊下去，感覺像是引導式的軍師。

**Comment by chupochun** [comment]
會不會在未來AI也有可能成為人類的真愛
